WalletWasabi.Fluent/Views/Wallets/MusicControlsView.axaml

```diff
@@ -70,7 +70,7 @@
 
           <DockPanel LastChildFill="False">
             <TextBlock Text="{Binding ElapsedTime}" />
-            <TextBlock Text="{Binding RemainingTime}" DockPanel.Dock="Right" />
+            <c:PrivacyContentControl NumberOfPrivacyChars="4" Content="{Binding RemainingTime}" DockPanel.Dock="Right" />
```

Comment:
> I'm just a bit concerned that it might hide the remaining time when privacy mode is activated? perhaps we can make something to prevent that?

For what I saw, the `RemainingTime` isn't time, but a BTC amount :) It's a bit counterintuitive!

Please, look at this:

https://github.com/SuperJMN/WalletWasabi/blob/3cc5dee2d723117074e1eb1e8e0824191a2df811/WalletWasabi.Fluent/ViewModels/Wallets/CoinJoinStateViewModel.cs#L313

Reply:
Oops, sorry @jmacato. I think you are referring to the fact that Remaining time is both an amount and a time depending on the status of the control!

I need to review that. Thanks!

Reply:
@jmacato I've updated the code. It doesn't rely on the PrivacyContentControl now, but in pure VM logic. Please, tell me your opinion. Thanks!

Reply:
> I don't prefer to not use the `PrivacyContentControl`. That is used everywhere and not using it in a place could introduce bugs later and also makes the code less readable.
> 
> I didn't check the code but if a variable with the name `RemainingTime` is used for displaying time and amount... that smells... so probably the displaying logic should be fixed and `PrivacyContentControl` should be used for hiding sensitive information.

I agree on all you say :) The fact that a property called `RemainingTime` is representing two completely different things is bit shocking.

But there's a point: we're using a control called `MusicControlsView`. It really makes sense for that context! So I guess there's some mismatch between the usage for which the controls was created and the usage we're giving to it in Wasabi.

I mean, it makes sense for a music control to have **remaining** and **elapsed** times.

As such, only 2 approaches come to my mind:

a) The current one. This is, keeping the control designed as is, and moving the privacy mode logic to the VM.
b) Modifying the current `MusicControlsView` to handly privacy like other parts in the UI. In this case, we should rename the control because it would no longer be a music control.

I would like @danwalmsley to take part to discuss how to proceed.
Thanks for your comments!